core/frontend/src/internal/render/webgl/BatchUniforms.ts

```diff
@@ -95,7 +95,7 @@ export class BatchUniforms {
   }
 
   public get wantContourLines(): boolean {
-    const contours = this._target.plan.contours;
+    const contours = this._target.is3d? this._target.plan.contours : this._target.currentContours;
```

Comment:
```suggestion
    const contours = this._target.currentContours;
```

Why the need to branch based on whether the view is 3d? You could have a branch in a 3d view that pushes different contour settings than those (if any) defined on the view.